src/content/getting-started/observability/logging/disable/index.md

```diff
@@ -0,0 +1,74 @@
+---
+linkTitle: Disable Managed Logging
+title: Disable Managed Logging for a Workload cluster
+description: This article explains how to turn off all managed logging for an entire workload cluster.
+weight: 50
+menu:
+  main:
+    identifier: getting-started-observability-logging-disable
+    parent: getting-started-observability-logging
+user_questions:
+  - How can I disable logging on my cluster?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2023-10-11
+aliases:
+  - /ui-api/observability/logging/disable
+  - /observability/logging/disable
+---
+
+In this article you will learn how you can disable logging for your cluster.
+
+## Introduction to logging
+
+Since releases 19.1.0, cluster created on the Giant Swarm platform benefits from our logging stack which allow us to provide you with 24/7 support to ensure best quality of service.
```

Comment:
We're introducing CAPI clusters, so we can't assume vintage releases or that the cloud provider is AWS.

Suggestion:

```
The managed logging stack allows Giant Swarm to provide 24/7 support based on your workload cluster logs. Currently, this component is pre-installed on workload clusters for the following implementations:

- {{% impl_title "vintage_aws" %}}, release [v19.1.0](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v19.1.0/) or newer
```